Sources/swift-test/usage.swift

```diff
@@ -35,7 +35,7 @@ enum Mode: Argument, Equatable, CustomStringConvertible {
         case "--help", "--usage", "-h":
             self = .Usage
         default:
-            return nil
+            self = .Run(argument)
```

Comment:
I think that with the removal of the `return nil` the initializer doesn't need to be failable and throwing anymore.